### ISSUE TITLE
Cannot Un-Reverse Pitch and Yaw

### ISSUE THREAD

**Issue by ghost** [MegaPirateNG/ardupilot-mpng#182]
I finally got the latest rel. of MPNG uploaded to my Crius AIO Pro v1.1 and while it connects to Mission Planner with no problems I've run into a nasty recurring issue with the radio inputs.

My radio gear is a Mode 2 setup using a FrSky DJT 8-channel Tx and FrSky FrSky V8FR-II Rx.

The problem: It took me several hours to get my quadcopter to respond correctly to the radio inputs, but once it was all working correctly I went into the Mission Planner GUI to calibrate the radio.

During calibration I noticed that Mission Planner was showing my pitch and roll inputs as reversed (roll was pitch and vice versa) so I exited Mission Planner and tested the quad only to discover that the pitch and roll had indeed become reversed.

I had no clue was was going on so I decided to try re-uploaded MPNG to the Crius -- amazingly this (somehow) resolved the issue.  At least momentarily.

After reinstalling MPNG  I went back into Mission Planner but this time I calibrated the radio using the CLI, not the GUI.  I then tested the copter and the controls were working correctly. Unfortunately it didn't last.

After my testing I disconnected the battery, waited a while, then re-connected the battery and performed another test. Sadly the pitch and roll had again become reversed!

I have no idea what's wrong or how to resolve this -- does anyone have some idea what's up with this?
Thanks!

**Comment by BQuad** [comment]
I'm having this exact issue. Were you ever able to resolve it?

**Comment by ghost** [comment]
No, I wasn't, but MPNG is kind of a lost cause now as it's no longer supported. I finally gave up and bought a Naze32 and installed CleanFlight.  Pretty awesome!  If you don't want to give up on your board I would suggest just using MultiWii.  It's not as robust as ArduCopter but at least it's still being updated.
